addons/website_forum/views/website_forum.xml

```diff
@@ -561,7 +561,7 @@
             data-toggle="tooltip"
             data-trigger="hover"
             title="My profile">
-            <img class="o_forum_avatar rounded-circle mr-1" t-att-src="website.image_url(user, 'image_128', '30x30')" alt="Avatar"/>
+            <img class="o_forum_avatar rounded-circle mr-1" t-att-src="website.image_url(request.env.user, 'image_128', '30x30')" alt="Avatar"/>
```

Comment:
use `user_id` which is the correct user and used by siblings lines
```suggestion
            <img class="o_forum_avatar rounded-circle mr-1" t-att-src="website.image_url(user_id, 'image_128', '30x30')" alt="Avatar"/>
```

Reply:
updated the pr as per the suggestion.